fastkml/kml.py

```diff
@@ -963,9 +963,11 @@ def point(self):
         return self._point
 
     @point.setter
-    def point(self, point):
-        if isinstance(point.geometry, Point):
-            self._point = point
+    def point(self, value):
```

Comment:
check that `point` is a `Point` instance?

Reply:
not worth the effort, see #154